src/ExtendedPdo.php

```diff
@@ -992,6 +992,23 @@ public function prepareWithValues($statement, array $values = array())
         return $sth;
     }
 
+
+    /**
+     *
+     * Set a specific rebuilder to handle the queries and returns the current rebuilder
+     *
+     * @param RebuilderInterface $rebuilder
+     *
+     * @return Rebuilder
+     *
+     */
+    public function setRebuilder(RebuilderInterface $rebuilder)
+    {
+        $oldRebuilder = $this->rebuilder;
+        $this->rebuilder = $rebuilder;
+        return $oldRebuilder;
```

Comment:
I don't think you need to return the old builder. What is the purpose of returning it?